TCMSApp/src/client/app/landing/landing.html

```diff
@@ -0,0 +1,248 @@
+<link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+<link rel="stylesheet" href="../../styles/landing-styles.css">
```

Comment:
are we using  less for this styles?

I think it can be the separate task if not. can you create? :)